client/landing/jetpack-cloud/index.js

```diff
@@ -16,11 +24,31 @@ import 'components/environment-badge/style.scss';
 import 'layout/style.scss';
 import 'assets/stylesheets/jetpack-cloud.scss';
 
+const debug = debugFactory( 'jetpack-cloud' );
+
+const boot = currentUser => {
+	debug( "Starting Jetpack Cloud. Let's do this." );
```

Comment:
Unneeded double quotes. Also, I vote for emoji. ⛈

Reply:
but I didn't want to have to escape the ' in `Let's`